wren-ai-service/src/utils.py

```
@@ -161,6 +161,24 @@ async def wrapper(*args, **kwargs):
     return wrapper
 
 
+def trace_cost(func):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)